go/ct/st/state_test.go

```diff
@@ -47,7 +47,7 @@ func getNewFilledState() *State {
 	s.Logs.AddLog([]byte{4, 5, 6}, NewU256(21), NewU256(22))
 	s.CallContext = CallContext{AccountAddress: vm.Address{0x01}}
 	s.BlockContext = BlockContext{BlockNumber: 1}
-	s.TransactionContext = TransactionContext{OriginAddress: vm.Address{0x01}}
+	s.TransactionContext = &TransactionContext{BlobHashes: []vm.Hash{{4, 3, 2, 1}}}
```

Comment:
Should this add `BlobHashes` besides the `OriginAddress` or is the replacement intentional?

Reply:
is intentional, since the idea is to change something in the structure, and in this case, changing `BlobHashes` is more interesting than an address.